PackageInfo.g

```diff
@@ -110,10 +106,18 @@ Status := "accepted",
 CommunicatedBy := "Charles R.B. Wright (Univ. of Oregon)",
 AcceptDate := "04/2003",
 
-README_URL := 
-  "http://www.algebra.uni-linz.ac.at/Sonata/README.sonata",
-PackageInfoURL := 
-  "http://www.algebra.uni-linz.ac.at/Sonata/PackageInfo.g",
+PackageWWWHome  := "https://gap-packages.github.io/sonata/",
+README_URL      := Concatenation( ~.PackageWWWHome, "README.md" ),
```

Comment:
This file does not exist. I suggest to convert it to .md seperately - whenever the website will move here or not, having it in .md will help to render it nicely on GitHub.

Reply:
Yeah, I copy & pasted the from `hecke` and forgot to adjust (the readme is in fact named `README.sonata` right now.